### DISCUSSION TITLE
商店过滤器中添加关键词但仍无法购买的问题

### DISCUSSION THREAD

**Discussion by shawnyuan-cn** [LmeSzinc/AzurLaneAutoScript#3063] [Q&A]
在按照[#3016](https://github.com/LmeSzinc/AzurLaneAutoScript/discussions/3016)的方法在过滤器中添加BoxIrisT2关键词后，经过一段时间观察发现仍无法自动购买鸢尾的蓝装备箱。
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/105029931/d3c4e97f-8ca2-4ec4-896d-6f2fbea2219d)

**Comment by LmeSzinc** [comment]
需要提供log和截图

**Reply by shawnyuan-cn** [reply]
**军火商这几次没刷出鸢尾的蓝装备箱，待几天再反馈**。
不过月初白票商店中的隐秘海域信息记录仪（HiddenZoneDataLogger）也会出现类似的情况，虽然按assets中的截图文件名添加到过滤器中，但并不能完成购买。
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/105029931/22d3436e-eba7-471d-a4d3-e647def9202f)
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/105029931/e409817f-7a02-4a87-8515-05ea0151301e)
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/105029931/816e74b1-7094-435e-9c6c-204e2f382d59)

**Comment by shawnyuan-cn** [comment]
1. 隐秘海域信息记录仪即使在手动购买后，也不会像效能样本一样在仓库中自动使用。（未验证，之前观察时印象中是这样的）
- **希望这东西能够像效能样本一样在仓库中检测到便会自动使用**。
- 理由是白票每月是溢出的，比起1000白票，希望能够始终开启隐秘海域的无视间隔（截图所示），节省开荒时间。
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/105029931/56f3b02e-73fb-461e-a930-3aa45d908cd0)

2. 在上面回复中的新材料“破碎的结晶”，也希望能在白票商店中自动购买，求一个关键词。

@LmeSzinc
